run.sh

```diff
@@ -48,22 +48,6 @@ if [ -z "$KEYSTORE_PATH" ] && [ -f "$(dirname $0)/certs/cryostat-keystore.p12" ]
     KEYSTORE_PASS="$(cat $(dirname $0)/certs/keystore.pass)"
 fi
 
-if [ ! -d "$(dirname $0)/archive" ]; then
-    mkdir "$(dirname $0)/archive"
-fi
-
-if [ ! -d "$(dirname $0)/conf" ]; then
-    mkdir "$(dirname $0)/conf"
-fi
-
-if [ ! -d "$(dirname $0)/conf/credentials" ]; then
-    mkdir "$(dirname $0)/conf/credentials"
```

Comment:
Where should `cryostat-user.properties` be stored? If I put the file in `conf/` or `conf/credentials` I get a 401 when logging in with the `BasicAuthManager`:
```
WARNING: User properties file "/opt/cryostat.d/conf.d/cryostat-users.properties" does not exist
Nov 15, 2021 3:33:49 PM io.cryostat.core.log.Logger warn
WARNING: HTTP 401: Unauthorized
```

Reply:
That's true, with this patch applied there is no longer anywhere to put a `cryostat-user.properties`. The `conf/` and `conf/credentials` in the working directory are no longer mounted into the container, nor is any other host directory mounted to the container at `/opt/cryostat.d/conf.d`.

I'll rethink this PR and see if there's a better and more compatible way to set up the mounts that doesn't result in access denied exceptions, while also giving us the flexibility of adding files like this.